4.analyze_data/notebooks/linear_model/0.fit_linear_model_strat.ipynb

```diff
@@ -19,7 +19,7 @@
     "from sklearn.linear_model import LinearRegression\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Assuming cp_df is your DataFrame
</pre><p>This comment may bring some confusion, since the code block does not show the usage of <code>cp_df</code>.  </p><p> </p><p>It seems the prior block make <code>X</code> as <code>cp_df</code> .   </p><p> </p><p>I would recommend updating either the comment or the variable name to improve readability. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/CFReT_data/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Whoops I didn't realize that code comment makes no sense, thanks for catching that! I will fix it!